configuration/customize-audit-logs-collection.md

```diff
@@ -132,3 +132,26 @@ To select Exchange activities:
 
 To view collected Exchange logs, use the **Exchange Logs report** available in the Report Center.
 
+
+
+# Audit Logs FAQ
```

Comment:
ovo staviti u H2
## Additional Information
ostale podnaslove ispod staviti u H3 ###
podnaslove reworkati da ne budu u stilu pitanja - npr. ##What type of audit logs does syskit point collect? promijeniti u npr. ###Collected Audit Logs